src/condor_io/condor_crypt.cpp

```diff
@@ -79,6 +81,12 @@ Condor_Crypto_State::Condor_Crypto_State(Protocol proto, KeyInfo &key) :
             m_ivec = (unsigned char*)malloc(m_ivec_len);
             break;
         }
+        case CONDOR_AESGCM: {
+            // ZKM TODO FIXME: CODE MISSING HERE TO INIT AES PROPERLY
```

Comment:
Clear up FIXME's prior to final merge.

I think this is already fixed, TBH.

Reply:
Yep, fixed.